includes/woopay/class-woopay-session.php

```diff
@@ -500,10 +503,6 @@ private static function get_woopay_verified_email_address() {
 	 * @return bool True if request is a Store API request, false otherwise.
 	 */
 	private static function is_store_api_request(): bool {
-		if ( ! defined( 'REST_REQUEST' ) || ! REST_REQUEST ) {
-			return false;
-		}
```

Comment:
Now that we've moved the initialization to a later point in the init process should we keep this check?

Reply:
This one I'm less sure on how to approach. This was also from @ricardo's fix so was curious if you think adding this conditional back in is adding protection or not given that we've moved the init call to now be later in the process when the constant should be defined?

Reply:
I think it's a valuable check it include, so I'd love to see us reintroduce this check if it doesn't break anything.

Reply:
I'm able to confirm that adding it back in prevents the currency lookup on WooPay from working. I believe this makes sense as we're still binding the `determine_current_user` hook during the `plugins_loaded` hook and `REST_REQUEST` isn't defined until `parse_request` which I believe comes sometime after `init`.

Reply:
Thank you for confirming, in that case I'll go ahead and approve ☺️